lib/client-assets.php

```diff
@@ -317,6 +317,52 @@ function gutenberg_extend_wp_api_backbone_client() {
 			} ) );
 		};
 JS;
+
+	/**
+	 * @var WP_REST_Server $wp_rest_server
+	 */
+	global $wp_rest_server;
+
+	// Ensure the rest server is initialized.
+	if ( empty( $wp_rest_server ) ) {
+		/** This filter is documented in wp-includes/rest-api.php */
+		$wp_rest_server_class = apply_filters( 'wp_rest_server_class', 'WP_REST_Server' );
+		$wp_rest_server       = new $wp_rest_server_class();
+
+		/** This filter is documented in wp-includes/rest-api.php */
+		do_action( 'rest_api_init', $wp_rest_server );
+	}
```

Comment:
You can use [`rest_do_request`](https://developer.wordpress.org/reference/functions/rest_do_request) instead of `$wp_rest_server->dispatch`, as done [elsewhere in the codebase](https://github.com/WordPress/gutenberg/blob/8253709173f0337ba7ea6ce1d39a32f18082b8ae/lib/client-assets.php#L344-L349).